### DISCUSSION TITLE
Im experiencing the error message: 'Session Start Failed'."SEB failed to start a new session! Please consult the log files

### DISCUSSION THREAD

**Discussion by nara989** [SafeExamBrowser/seb-win-refactoring#862] [Q&A] [answered]
Based on the client.log I read, it seems that the issue lies here:
   Exception Message: A procedure imported by 'CefSharp.Core.Runtime.dll' could not be loaded.
   Exception Type: System.IO.FileLoadException

Despite using the Latest Microsoft Visual C++ Redistributable Version, which is version 14.38.33135.0, and attempting a reinstall, the issue persists. The client.log still indicates:
Exception Message: A procedure imported by 'CefSharp.Core.Runtime.dll' could not be loaded.
   Exception Type: System.IO.FileLoadException

This is the client file and the runtime:
[2024-04-25_14h20m11s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/15106083/2024-04-25_14h20m11s_Client.log)
[2024-04-25_14h20m11s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/15106084/2024-04-25_14h20m11s_Runtime.log)

**Comment by dbuechel** [comment]
Try installing the version specified in the release notes, namely the Visual C++ 2015-2019 Redistributable: https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads.

**Reply by nara989** [reply]
I've tried your suggestion, but this error still occurs and the details are still showing in the client.log.

![seb](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/168071613/0c4e89d5-7dfc-4e25-8524-450e1f557c25)

![VC++ 2015-2019](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/168071613/ec55a2bf-f5fe-4a79-a4c5-3bd6adc65e33)

This is the link to the VC++ 2015-2019 that I downloaded:
https://downloads.i-theses.com/index.php?option=com_downloads&Itemid=101&task=downloads&groupid=9&id=124

This is the client file and the runtime:
[2024-04-25_20h50m22s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/15111214/2024-04-25_20h50m22s_Client.log)
[2024-04-25_20h50m22s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/15111283/2024-04-25_20h50m22s_Runtime.log)

**Comment by dbuechel** [comment, marked as answer]
Terribly sorry, just now realized that you're using Windows 7. That operating system version is not supported anymore, neither by us nor the browser engine manufacturer, and thus also the reason of failure: https://github.com/cefsharp/CefSharp/discussions/4336.

**Reply by nara989** [reply]
Thanks for the explanation. Very helpful. So I have to update my Windows system to Windows 10 and above before I can run the application.

**Reply by dbuechel** [reply]
Yes exactly, I'm afraid there is no way around it.
